src/config/backpack/base.php

```diff
@@ -14,6 +14,7 @@
     // Date & Datetime Format Syntax: https://carbon.nesbot.com/docs/#api-localization
     'default_date_format'     => 'D MMM YYYY',
     'default_datetime_format' => 'D MMM YYYY, HH:mm',
+    'default_time_format'     => 'hh:mm a',
```

Comment:
I think a better default would be "_military time_" - as far as I know, most of the globe uses this one. This also bring it in line with the `default_datetime_format`, which uses this one too:

```suggestion
    'default_time_format'     => 'HH:mm',
```